website/client/src/store/getters/worldState.js

```diff
@@ -0,0 +1,29 @@
+export function brokenSeasonalShop (store) {
+  const worldState = store.state.worldState.data;
+
+  return worldState
+    && worldState.worldBoss
+    && worldState.worldBoss.extra
+    && worldState.worldBoss.extra.worldDmg
+    && worldState.worldBoss.extra.worldDmg.seasonalShop;
```

Comment:
Perhaps use https://lodash.com/docs/4.17.15#get instead? So that would be something like:

```suggestion
  return lodash.get(worldState, 'worldBoss.extra.worldDmg.seasonalShop');
```

Reply:
It's a bit nicer but I don't think it's worth introducing the dependency. Also in future we should be able to use this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Elvis is indeed the modern way to do this. Perhaps introduce a private method along the lines of `function getWorldDamage(worldState, type)` to reduce the duplication?

Reply:
Fixed in https://github.com/HabitRPG/habitica/pull/12179/commits/a89c436c400be7d862480b84ac0f4ba2c14714aa